lib/engine/game/g_1854/step/merge_minors.rb

```diff
@@ -0,0 +1,114 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/base'
+
+module Engine
+  module Game
+    module G1854
+      module Step
+        class MergeMinors < Engine::Step::Base
+          def actions(entity)
+            return [] if entity.corporation? && !entity.minor?
+            return [] if !entity.minor? || entity != current_entity #  || @game.done_this_round[entity]
+            return [] unless @game.mergeable?(entity)
+            return [] if target_corporations.empty?
+            return [] if @merge_target
+
+            actions = []
+            actions << 'merge' if @game.minor_mergers_allowed?
+            actions << 'pass' if @game.minor_mergers_allowed? && !@game.minor_mergers_required?
+            actions
+          end
+
+          def blocking?
+            @game.mergeable?(current_entity)
+          end
+
+          def active?
+            return false if @game.open_minors.empty?
+
+            @game.minor_mergers_allowed? || @game.minor_mergers_required?
+          end
+
+          def setup
+            @mergeable_ = {}
+            @merging = nil
+            @merge_target = nil
+          end
+
+          def auto_actions(entity)
+            return super if @merging
+
+            return [Engine::Action::Pass.new(entity)] if mergeable_candidates(entity).empty?
+
+            super
+          end
```

Comment:
I'm fairly certain that you don't need auto_actions here. Is this copied from another game? You'd need this where you have to check the merge candidates are connected by track, but want to avoid recalculating the graph when a game is being loaded. In 1854 there is no requirement for a track connection between the local railways, so this is not an issue.

Reply:
This is the general flow for this step which i copied from one of 1841, 1862, 1867, 18ESP, 18NewEngland.  They all use basically the same logic for merging, and all of those have this same auto-action.  I didn't fully understand the intricacies re: track connection and stuff.  I will delete this, and there's still a long way to do and testing to do.